docs/src/reference/gremlin-variants.asciidoc

```diff
@@ -682,6 +683,14 @@ This command will generate a new Maven project in a directory called "app" with
 `com.my`. Please see the `README.asciidoc` in the root of each generated project for information on how to build and
 execute it.
 
+[[gremlin-java-user-agent]]
+=== User Agent
```

Comment:
i wouldn't make this a separate top-level section as GLVs have a set form. i was going to suggest you could make it a fourth level section under `gremlin-java-configuration` but i feel like this the "user agent" is just going to then have to be replicated for each GLV. I'm starting to think about this differently now. This new use of user agent is technically a standard that drivers should adhere to. I think you should document it as such in the [Provider Documentation](https://github.com/apache/tinkerpop/blob/master/docs/src/dev/provider/index.asciidoc#graph-driver-provider-requirements) where you should include the expected format, examples, etc. Then you can just add a link to that from your little configuration description if folks want to know what the "use agent" is. Finally in the Upgrade Documentation you should make mention of this new feature in the Provider section to call attention to it. how does that sound?